docs/spl-docs/evaluation-functions/multivalue-functions.md

```diff
@@ -35,4 +35,198 @@ The following command returns the split parts of the given string as a multivalu
 2. The `split` function is used to divide the `recipients` string into separate email addresses based on the semicolon (`;`) delimiter.
   - For example, if `recipients` contains `user1@example.com;user2@example.com;user3@example.com`, the `split` function will create an `email_list` multivalue field with three separate values: `user1@example.com`, `user2@example.com`, and `user3@example.com`.
 
-This method allows for detailed analysis of email logs, such as calculating response rates or identifying recipient engagement, by treating each recipient address as an individual entity.
\ No newline at end of file
+This method allows for detailed analysis of email logs, such as calculating response rates or identifying recipient engagement, by treating each recipient address as an individual entity.
+
+## mvindex(\<mv\>, \<start\>, \<end\>)
+
+This function extracts a specific subset of values from a multivalue field based on the provided start and end index positions.
+
+### Usage
+- The `<mv>` parameter must be a multivalue field and `<start>` and `<end>` must be numbers.
+- The `<start>` parameter is required and indicates the starting index of the value(s) you want to extract.
+- The `<end>` parameter is optional. If provided, it defines the ending index, and the function will return all values between `<start>` and `<end>`, inclusive.
+
+### Indexing Details
+- Indexes begin at 0, meaning the first item in a list is at position 0, the second at position 1, and so on.
+- If only `<start>` is specified, the function returns the value at that position.
+- If both `<start>` and `<end>` are specified, the function returns all values from `<start>` to `<end>`.
+- Negative indexes are supported, where `-1` refers to the last value in the list, `-2` to the second last, and so forth.
+- If the indexes provided are out of range or invalid, the function returns `NULL`.
+
+### Example
+Suppose you have a multivalue field called `cities` containing the following values:
+
+```
+New York, London, Paris, Tokyo, Sydney, Berlin, Dubai
+```
+
+To extract the third city in the list (Paris), use:
+
+```
+... | eval selected_city=mvindex(cities,2)
+```
+
+This command will set `selected_city` to `Paris`.
```

Comment:
Does this return a multivalue value with one item, or just a normal value (so there's only one item)?